contracts/interfaces/IQuestFactory.sol

```diff
@@ -84,6 +84,7 @@ interface IQuestFactory {
         uint16 referralFee,
         uint256 mintFeeEthWei
     );
+    event MintFeePaid(address referrer, uint256 referrerAmount, address mintFeeRecipient, uint256 mintFeeRecipientAmount);
```

Comment:
let's do `referrerAmountWei` and `mintFeeRecipientAmountWei`

Reply:
I think questId will be needed in order for us to track earning per quest per creator